docs/REST API/rest-api.md

```diff
@@ -1970,6 +1970,7 @@ the `software` table.
 | mdm_id                  | integer | query | The ID of the _mobile device management_ (MDM) solution to filter hosts by (that is, filter hosts that use a specific MDM provider and URL).                                                                                                                                                                                                |
 | mdm_name                | string  | query | The name of the _mobile device management_ (MDM) solution to filter hosts by (that is, filter hosts that use a specific MDM provider).                                                                                                                                                                                                |
 | mdm_enrollment_status   | string  | query | The _mobile device management_ (MDM) enrollment status to filter hosts by. Valid options are 'manual', 'automatic', 'enrolled', 'pending', or 'unenrolled'.                                                                                                                                                                                                             |
+| connected_to_fleet   | string  | query | Filter hosts that are connected and controlled by your Fleet server. Some hosts can be enrolled in MDM solution "Fleet" but pointing to other Fleet servers and not yours.                                                                                                                            |
```

Comment:
@noahtalerman Wdyt about this?

Reply:
```suggestion
| mdm_this_fleet   | string  | boolean | Filter hosts that talk to this Fleet server for MDM features. In some rare cases, hosts can be enrolled to one Fleet server but talk to a different Fleet server for MDM features.                                                                                        |
```

Hmm, I think `connected_to_fleet` could be misleading. This will always set to `null` for Linux hosts, right? Aren't they connected to Fleet?

What do you think about my suggestion above? This is assuming this filter returns a list of macOS, Windows, and iOS/iPadOS hosts that are talking to Fleet for MDM features.

Also, this property should be a boolean right?

cc @rachaelshaw and @roperzh

Reply:
good point, this is a boolean. Heads-up that this is shipping today as `connected_to_fleet`

Reply:
Ah, ok. I don't think this needs to block shipping the release.

> This will always set to null for Linux hosts, right? Aren't they connected to Fleet?

> assuming this filter returns a list of macOS, Windows, and iOS/iPadOS hosts that are talking to Fleet for MDM features.

But, if the above is right, then I think let's track a bug/story for a quick follow up to the naming. I think let's also wait to merge this PR so that we can break the API.

cc @rachaelshaw